src/libs/fileDownload/index.ts

```diff
@@ -8,9 +8,10 @@ import type {FileDownload} from './types';
 /**
  * The function downloads an attachment on web/desktop platforms.
  */
-const fileDownload: FileDownload = (url, fileName) => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why is this disable needed?

Reply:
@allroundexperts we require `shouldOpenExternalLink` but are not utilizing `successMessage` there, which could lead to a lint error.